nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/main/java/org/nuxeo/elasticsearch/audit/pageprovider/ESAuditPageProvider.java

```diff
@@ -175,16 +177,43 @@ protected ESAuditBackend getESBackend() {
     protected void buildAuditQuery(boolean includeSort) {
         PageProviderDefinition def = getDefinition();
         Object[] params = getParameters();
+        List<QuickFilter> quickFilters = getQuickFilters();
+        String quickFiltersClause = "";
+
+        if (quickFilters != null && !quickFilters.isEmpty()) {
+            for (int i = 0; i < quickFilters.size(); i++) {
+                if (!quickFiltersClause.isEmpty() && quickFilters.get(i).getClause() != null) {
```

Comment:
You can put the `clause` (quickFilters.get(i).getClause()) in a local variable as you use it several times.